finetune_openai_models/README.md

```diff
@@ -16,3 +16,23 @@ Create a file called `Config.toml` at the root of the project.
 ```
 openAIToken="<OPENAI_API_KEY>"
 ```
+
+### Preparing training data jsonl file
+The training data jsonl file should contain `prompt` and `completion` pairs for each sample in each line. An example is added in the data directory. Please refer `prepare training data` section in https://platform.openai.com/docs/guides/fine-tuning for more information.
+
+### Obtaining predictions from the fine-tuned model
+Generating predictions from the fine-tuned model can be done similar to any base model provided by OpenAI. You just have to specify the model id of the fine-tuned model in the `model` parameter of the request. You can find the model id of the fine-tuned model by doing an API call to `openai.finetunes` or by simply logging into the playground of your OpenAI account.
```

Comment:
Don't we need to give an example on how to do this. `fine-tuned model by doing an API call to `openai.finetunes``

Reply:
Shall we add the playground link for this `logging into the playground`

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/58/commits/7ed3aaa7d8d758e1ca7b378d74f1495f22553c07.